### PR TITLE
avoid race condition in jquery initialisation

### PR DESCRIPTION
**Pull request by pbruski**

### CODE REVIEW

File: src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/DashboardPage.kt [hunk 1]
```diff
@@ -46,6 +46,9 @@ class DashboardPage(
             //we currently support only single iframe on dashboard in this check
             return input.executeScript(
                 """
+                if (typeof $ === 'undefined') { //wait for jquery
+                    return false;
+                }
                 iframes = $('#dashboard').find('iframe');
```
Comment (pbruski) [inline src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/DashboardPage.kt]: This causes about 5% failure rate of dashboard view in our tests

Comment (wyrzyk) [reply src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/DashboardPage.kt]: Nice finding!

**Comment by pbruski** [comment]
Rebased to remove merge commit.

File: src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/DashboardPage.kt (hunk 1 above)
Comment (dagguh) [reply src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/DashboardPage.kt]: Whoah!

**Pull request merged**